widgetssdk/src/main/java/com/glia/widgets/call/CallActivityIntentHelper.kt

```diff
@@ -0,0 +1,41 @@
+package com.glia.widgets.call
+
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import com.glia.androidsdk.Engagement
+import com.glia.widgets.GliaWidgets
+import com.glia.widgets.core.configuration.GliaSdkConfiguration
+
+class CallActivityIntentHelper {
```

Comment:
I think you forgot to make it internal, also it's better to convert it to `object` instead of using `companion object`.

Reply:
Fixed. Also please check the end result again. I also have added one more class into the mix of classes converted to Kotlin